sdk/search/CONTRIBUTING.md

```diff
@@ -72,57 +72,54 @@ live tests against a pull request by commenting `/azp run net - search - tests`
 in the PR.
 
 ### Live Test Resources
-You'll need to pass the details for a subscription, service principal, resource
-group, and location via environment variables.  We set this up for live tests
-using the [test-resources.json](./test-resources.json) ARM template, but it's
-equivalent to the following steps if you want to run the live tests locally.
+To set up your Azure account to run live tests, you'll need to log into Azure,
+create a service principal, and set up your resources defined in
+[test-resources.json](./test-resources.json) as shown in the following example.
+Note that `-Subscription` is an optional parameter but recommended if your account
+is a member of multiple subscriptions.
 
 ```powershell
-PS C:\src> az ad sp create-for-rbac --sdk-auth
-Creating a role assignment under the scope of "/subscriptions/<subscription_id>"
-{
-  "clientId": "<client_id>",
-  "clientSecret": "<client_secret>",
-  "subscriptionId": "<subscription_id>",
-  "tenantId": "<tenant_id>",
-  "activeDirectoryEndpointUrl": "https://login.microsoftonline.com",
-  "resourceManagerEndpointUrl": "https://management.azure.com/",
-  "activeDirectoryGraphResourceId": "https://graph.windows.net/",
-  "sqlManagementEndpointUrl": "https://management.core.windows.net:8443/",
-  "galleryEndpointUrl": "https://gallery.azure.com/",
-  "managementEndpointUrl": "https://management.core.windows.net/"
-}
-
-PS C:\src> az group create --location <location> --name <resource_group>
-{
-  "id": "/subscriptions/<subscription_id>/resourceGroups/<resource_group>",
-  "location": <location>,
-  "managedBy": null,
-  "name": "<resource_group>",
-  "properties": {
-    "provisioningState": "Succeeded"
-  },
-  "tags": null,
-  "type": "Microsoft.Resources/resourceGroups"
-}
-
-PS C:\src> setx AZURE_TENANT_ID <tenant_id>
-SUCCESS: Specified value was saved.
-
-PS C:\src> setx AZURE_CLIENT_ID <client_id>
-SUCCESS: Specified value was saved.
-
-PS C:\src> setx AZURE_CLIENT_SECRET <client_secret>
-SUCCESS: Specified value was saved.
-
-PS C:\src> setx AZURE_SUBSCRIPTION_ID <subscription_id>
-SUCCESS: Specified value was saved.
-
-PS C:\src> setx AZURE_RESOURCE_GROUP <resource_group>
-SUCCESS: Specified value was saved.
-
-PS C:\src> setx AZURE_LOCATION <location>
-SUCCESS: Specified value was saved.
+PS C:\src> Connect-AzAccount -Subscription 'YOUR SUBSCRIPTION ID'
```

Comment:
Are the PS cmdlets required for this?  We've preferred using the CLI for all of our samples.

Reply:
That's changing with the new system we're starting to roll out. I'll be linking to a central doc (offline conversation) and will include a blurb about installing them.

Reply:
Part of the problem was because of automating acquisition and cross-platform scripting of the azure CLI turned up some issues. Using the azure CLI was my original intent, but to have a cross-platform scripting interface - pwsh - was needed anyway, acquisition of the azure CLI was not only way slower (seems to keep a copy of python in it, and tons of tiny files take forever to extract), but deemed extraneous when we can just get the module from within PowerShell we have to use already. This was a cross-team decision.